src/main/resources/edu/cornell/kfs/module/purap/cu-ojb-purap.xml

```diff
@@ -1141,7 +1153,10 @@
         <field-descriptor name="vendorAttentionName" column="VNDR_ATTN_NM" jdbc-type="VARCHAR"/>
         <field-descriptor name="assignedUserPrincipalId" column="PO_ASND_ID" jdbc-type="VARCHAR"/>
         <field-descriptor name="accountDistributionMethod" column="ACCT_DISTR_MTD_IND" jdbc-type="VARCHAR"/>
-        
+        <reference-descriptor name="vendorCountry" class-ref="org.kuali.kfs.sys.businessobject.Country"
+                              auto-retrieve="true" auto-update="none" auto-delete="none" proxy="true">
+            <foreignkey field-ref="vendorCountryCode" target-field-ref="code"/>
+        </reference-descriptor>
```

Comment:
It looks like there are two "vendorCountry" references in the PurchaseOrderReopenDocument descriptor, and a "vendorCountry" reference appears to be missing from the CuPurchaseOrderAmendmentDocument descriptor. Can you please update these accordingly to have one "vendorCountry" reference each?